docs/src/getting_started/toolsinstallation.md

```diff
@@ -1 +1,110 @@
 # [Installing Development Tools](@id toolsinstallation)
+
+## [Julia REPL](@id julia-repl)
+
+A REPL, or read-eval-print loop, represents a language "shell" or program that allows the user to communicate with the
+programming language interactively with immediate feedback. This allows for exploring the language more deeply than via
+a script, with rapid prototyping being much quicker and easier to perform.
+
+The Julia REPL is included by default in every Julia installation and boasts a number of features to the point that it
+alone can almost be considered a proper IDE. It should appear the moment you enter `julia` into your terminal:
+
+```
+   _       _ _(_)_     |  Documentation: https://docs.julialang.org
+  (_)     | (_) (_)    |
+   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
+  | | | | | | |/ _` |  |
+  | | |_| | | | (_| |  |  Version 1.9.2 (2023-07-05)
+ _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
+|__/                   |
+
+julia> 
+```
+
+We won't go into every feature that the REPL provides, however here is a quick list that most Julia users should
+know to make the most of this shell:
+
+- There are 5 Julia REPL modes that serve separate purposes: 
+  1. The Julian (default) mode wherein one can write, load, and test out code
+  2. Help mode for printing documentation for functions, keywords, etc.
+  3. Pkg mode for working with the included package manager and manipulating dependencies/packages
+  4. Shell mode to execute system commands (run a terminal from Julia which itself is running from a terminal!) 
+  5. Search modes (forward, reverse) to look through your history of inputs
+
+|        Prompt         |         Key binding         |
+|:---------------------:|:---------------------------:|
+|       `julia> `       |     None (default mode)     |
+|       `help?> `       | `?` or `??` (extended help) |
+|    `(@v1.x) pkg> `    |             `]`             |
+|       `shell> `       |             `;`             |
+| `(reverse-i-search):` |            `^R`             |
+| `(forward-i-search):` |            `^S`             |
+
+- If you have a code editor installed on your development machine, you can go back and forth between the Julia session
+  and the editor by entering `julia> edit("<name of file>")`. You can jump right back into the Julia session once you're 
+  finished!
+- You can get a list of matches to a partially written word by entering the TAB key, which also serves to render an
+  Unicode symbol in the REPL (full list available in the [Unicode Input](https://docs.julialang.org/en/v1/manual/unicode-input/) section of the Julia manual): 
+  ```julia
+  julia> pri[TAB]
+  primitive type  print           println         printstyled
+  julia> \pi[TAB] # renders π
+  ```
+- You can run a standalone script by entering `include("<name of script>")` in Julian mode. For greater control, the 
```

Comment:
````suggestion
  Unicode symbol in the REPL (full list available in the [Unicode Input](https://docs.julialang.org/en/v1/manual/unicode-input/) section of the Julia manual): 

    ```julia
  julia> pri[TAB]
  primitive type  print           println         printstyled
  julia> \pi[TAB] # renders π
  ```

- You can run a standalone script by entering `include("<name of script>")` in Julian mode. For greater control, the 
````

More new-lines between things, in general